### DISCUSSION TITLE
Anyway to create both One File and One Folder with the same command?

### DISCUSSION THREAD

**Discussion by TheWicklowWolf** [pyinstaller/pyinstaller#7610] [Help] [answered]
As above, would like to create both one file and one folder at the same time..

**Comment by rokm** [comment]
Not directly. 

But if you really want both at the same time, you could probably merge a one-file and a one-dir spec file into new one (with shared analysis and pyz, with onefile exe, and onedir exe+collect) and build using the combined spec file.

**Comment by TheWicklowWolf** [comment, marked as answer]
Just used a batch file. Thanks!
